schemas/commodity-README.md

```diff
@@ -0,0 +1,89 @@
+# EDDN Commodity Schema
+
+## Introduction
+Here we document how to take data from an ED `Market` Journal Event and
+properly structure it for sending to EDDN.
+
+Please consult [EDDN Schemas README](./README-EDDN-schemas.md) for general
+documentation for a schema such as this.
+
+## Senders
+The primary data source for this schema is the ED Journal event `Market`, 
+and the additional file, `Market.json`, that it signals the writing of.
+
+So, look for the `Market` event, and when it occurs open and read the 
+`Market.json` file which contains the actual data.  Treat *that* data as 
+the event.
+
+It *is* also possible to construct this data from a CAPI `/market` query.  
+See [Using CAPI data](#using-capi-data) below.
+
+### Key Renames
+Many of the key names have a different case defined in this schema, make 
+sure you are renaming them as appropriate.
+
+#### StarSystem to systemName
+Rename the `StarSystem` key name to `systemName`.
+
+### Elisions
+#### Remove _Localised key/values
+All keys whose name ends with `_Localised`, i.e. the `Name_Localised`
+key/values in Items.
+
+#### Other Elisions
+You MUST remove the following key/value pairs from the data:
+
+- `StationType` key/value.
+- `Producer` key/value pair in Items.
+- `Rare` key/value pair in Items.
+- `id` key/value pair in Items.
+
+In the list of commodites:
+
+- Skip commodities with `"categoryname": "NonMarketable"` (i.e.
+  Limpets - not purchasable in station market) or a *non-empty*`"legality":` 
+  string (not normally traded at this station market).
+
+#### Item Category
+Remove not only the `Category_Localised` key/value, as above, but also the
+`Category` key/value pair from each Item.
+
+### Augmentations
```

Comment:
`statusFlags` is an optional augmentation that can be used to identify `Producer`, `Consumer`, and `Rare` goods.

Reply:
I was leaving this as implied, as it's in the schema.  I can add something based on your comment though.